clang/test/SemaSYCL/check-notdirect-attribute-propagation.cpp

```diff
@@ -5,7 +5,7 @@
 #ifndef TRIGGER_ERROR
 [[intel::no_global_work_offset]] void not_direct_one() {} // expected-no-diagnostics
 
-[[intel::reqd_sub_group_size(1)]] void func_one() {
+[[intel::reqd_sub_group_size(2)]] void func_one() {
```

Comment:
Is this change intentional?

Reply:
No, this is not intentional. Test is changed to original.